File: 4-median-of-two-sorted-arrays/median-of-two-sorted-arrays.cs
public class Solution {
    public double FindMedianSortedArrays(int[] nums1, int[] nums2) {
        int[] nums = new int[nums1.Length + nums2.Length];
        Array.Copy(nums1, 0, nums, 0, nums1.Length);
        Array.Copy(nums2, 0, nums, nums1.Length, nums2.Length);
        Array.Sort(nums);

        if (nums.Length % 2 != 0) {
            return (double)nums[nums.Length / 2];
        }
        return ((double)nums[(nums.Length - 1) / 2] + nums[nums.Length / 2]) / 2.0;
    }
}

File: 4-median-of-two-sorted-arrays/median-of-two-sorted-arrays.java
class Solution {
    public double findMedianSortedArrays(int[] nums1, int[] nums2) {
        int[] nums = new int[nums1.length + nums2.length];
        System.arraycopy(nums1, 0, nums, 0, nums1.length);
        System.arraycopy(nums2, 0, nums, nums1.length, nums2.length);
        Arrays.sort(nums);
        if(nums.length % 2 != 0){
            return (double)nums[nums.length/2];
        }
        return (double)(nums[(nums.length - 1) / 2] + nums[nums.length / 2]) / 2.0;
    }
}